sklearn/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''

```

Comment:
Use double-quote rather than single-quote which is more conventional for docstring.